test/python/quantum_info/operators/channel/test_evolve.py

```diff
@@ -41,10 +42,9 @@ def _unitary_to_other(self, rep, qubits_test_cases, repetitions):
             for _ in range(repetitions):
                 rho = self.rand_rho(dim)
```

Comment:
There was some discussion about using random in tests. Did we come to an agreement about that?

Reply:
I'm not aware of the outcome of that discussion, but I can say I found the random tests quite helpful for finding bugs

Reply:
More or less, the outcome was that tests that depend on randomness should set a seed, use a `RandomState`, and log the seed (see #1863). If you have tests that are intended to run continuously to look for bugs, they should live in `test/randomized`.

Reply:
Added logging as per example in 1863